recipe/conda_build_config.yaml

```diff
@@ -1,2 +0,0 @@
-channel_sources:
-  - conda-forge,conda-forge/label/dev,defaults
```

Comment:
this is actually necessary since we need to pick a requirement from a `dev` label.